Update slashing-protection interchange format version to v.5.

rolfyone opened the pull request:
fixes #3197

Signed-off-by: Paul Harris <paul.harris@consensys.net>

## Documentation

- [X] I thought about documentation and added the `documentation` label to this PR if updates are required.

rolfyone committed:
Update slashing-protection interchange format version to v.5.

The difference is 'interchange_format' is no longer required. If you have v.4 and want to import it with v.5, delete that field from metadata, and update the version.

fixes #3197

Signed-off-by: Paul Harris <paul.harris@consensys.net>

rolfyone committed:
Merge remote-tracking branch 'upstream/master' into 3197

ajsutton commented on data/dataexchange/src/main/java/tech/pegasys/teku/data/SlashingProtectionImporter.java:
```diff
@@ -81,6 +81,9 @@ public void initialise(final File inputFile) throws IOException {
       String cause = e.getCause() != null ? e.getCause().getMessage() : e.getMessage();
+      if (cause.contains("\"interchange_format\"")) {
+        cause = "Required version is " + INTERCHANGE_VERSION;
+      }
```
This doesn't seem ideal. If the only change is removing this field, can we support both version 4 and 5 and just mark the field optional?

rolfyone committed:
make interchange-format optional rather than removing it.

Signed-off-by: Paul Harris <paul.harris@consensys.net>

rolfyone committed:
make interchange-format optional rather than removing it.

allow v.4 or v.5 to be taken in, it's only the interchange_format that's been removed.

Signed-off-by: Paul Harris <paul.harris@consensys.net>

rolfyone committed:
Merge remote-tracking branch 'upstream/master' into 3197

ajsutton reviewed (approved):
LGTM.
